src/System.Diagnostics.TraceSource/src/System/Diagnostics/DefaultTraceListener.cs

```diff
@@ -115,7 +115,34 @@ private void WriteAssert(string stackTrace, string message, string detailMessage
         /// </devdoc>
         public override void Write(string message)
         {
-            if (NeedIndent) WriteIndent();
+            Write(message, true);
```

Comment:
Nit: for cases where Boolean arguments are passed, or other non-descript values, it's helpful where possible to use parameter names at the call site, e.g.
```C#
Write(message, useLogFile: true);
```